lib/improver/lapse_rate.py

```diff
@@ -107,7 +107,7 @@ class LapseRate(object):
     4) Loop through array of neighbourhoods and take the height and temperature
        of all grid points and calculate the
        temperature/height gradient = lapse rate
-    5) Constrain the lapse rate as > DALR and < -3.0*DALR.
+    5) Default constraints for the lapse rate set as > DALR and < -3.0*DALR.
```

Comment:
It's worth point this out, I think we just need to ensure that we are still stating the final step in the process here.

```suggestion
    5) Constrain the returned lapse rates between min_lapse_rate and max_lapse_rate. These default to > DALR and < -3.0*DALR but are user configurable.
```

Reply:
Docstring updated